certified-connectors/Inserve/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "connectionParameters": {
+        "subdomain": {
+            "type": "string",
+            "uiDefinition": {
+              "displayName": "subdomain",
+              "description": "The subdomain from your Inserve environment.",
+              "tooltip": "Host subdomain",
+              "constraints": {
+                "clearText": true,
+                "required": "true"
+              }
+            }
+          },
+          "api_key": {
+            "type": "securestring",
+            "uiDefinition": {
+              "displayName": "API Key",
+              "description": "The API Key for this api",
+              "tooltip": "Provide your API Key",
+              "constraints": {
+                "clearText": false,
+                "required": "true"
+              }
+            }
+          }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Frank Derks",

```

Comment:
@frank-inserve since you confirmed that you are a verified publisher. please change this to the company name.